sdk/opa.go

```diff
@@ -300,7 +300,7 @@ func (opa *OPA) executeTransaction(ctx context.Context, record *server.Info, wor
 
 	if record.Error == nil {
 		defer s.manager.Store.Abort(ctx, record.Txn)
-		work(result)
+		work(s, result)
```

Comment:
Should this not be `state` as that's what the argument name is?

Reply:
The argument to `executeTransaction` is a function of type `func(state state, result *DecisionResult))` _named_ `work`. So this is OK.

Reply:
Ah, thanks 👍